typescript/cli/src/config/chain.ts

```diff
@@ -111,6 +114,42 @@ export async function createChainConfig({
   }
 }
 
+async function addBlockExplorerConfig(metadata: ChainMetadata): Promise<void> {
+  const wantBlockExplorerConfig = await confirm({
+    default: false,
+    message: 'Do you want to add a block explorer config for this chain',
```

Comment:
what style are we going for in the cli, should we phrase this as "Do you want to configure a block explorer for this chain"?

Reply:
i'm alright with both, this was to emulate the current format in the other config functions.

i've added you comment here to capture in the future: https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/3770#issuecomment-2231431779